src/amo/addonManager.js

```diff
@@ -42,9 +42,9 @@ type PrivilegedNavigatorType = {|
   mozAddonManager: MozAddonManagerType,
 |};
 
-type OptionalParams = {|
+type OptionalParams = {
```

Comment:
When we do need to provide a deafult of `{}` the alternative solution is to define the type of the params as inexact, which is what I have done here, and elsewhere.